clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/TransferReadGroupV1QueryOutput.scala

```diff
@@ -3,15 +3,53 @@ package org.broadinstitute.clio.transfer.model
 import java.time.OffsetDateTime
 
 case class TransferReadGroupV1QueryOutput(
```

Comment:
I could version the currently unversioned names if that's better.
Some classes are not versioned because they are always the most recent view of the data.